roxctl/netpol/generate/generate.go

```diff
@@ -80,8 +82,20 @@ func (cmd *netpolGenerateCmd) construct(args []string, c *cobra.Command) (*npgua
 	cmd.inputFolderPath = args[0]
 	cmd.splitMode = c.Flags().Changed("output-dir")
 	cmd.mergeMode = c.Flags().Changed("output-file")
+	dnsPortNum, err := strconv.Atoi(cmd.Options.DNSPort)
+	if err == nil {
+		cmd.dnsPortNum = dnsPortNum
```

Comment:
User could provide here a negative number and we would accept it. I think, we need to add some validation.

Reply:
This should be caught in `validate()`